compiler/src/dotty/tools/dotc/transform/PostTyper.scala

```diff
@@ -203,7 +203,9 @@ class PostTyper extends MacroTransform with IdentityDenotTransformer { thisPhase
               // might be a type constructor.
               Checking.checkInstantiable(tree.tpe, nu.pos)
               withNoCheckNews(nu :: Nil)(super.transform(tree))
-            case _ =>
+            case meth =>
+              if (meth.symbol.name.eq(nme.QUOTE) && meth.symbol.owner.eq(defn.OpsPackageClass))
```

Comment:
Just `meth.symbol.isQuote` should do. But I don't see why this is needed. Does `handleMeta` in the `Select` case earlier not catch this? Or maybe we need a `handleMeta` for `Ident`s as well?

Reply:
This is half of the fix in #3777. I will explore the different solutions in that PR.

Reply:
`handleMeta` seems to be the simplest solution. I will merge #3777 into this PR.